### DISCUSSION TITLE
MigrationPlan per doctype

### DISCUSSION THREAD

**Discussion by giuunit** [Jumoo/uSyncMigrations#126] [Q&A] [answered]
First of all, I wanna say, I love this project, great idea, great concept and the doc is really great. I do have one question about a use-case on our side. 

We are migrating a project from v7 with vorto, grid and nested content to v11 with native language variant properties, BGE and BLE. 

During the development phase of our project, we decided to change the data type of some properties depending on the doc type, example: 

* Landing page in v7 had a nested content data type property that is now a BLE
* Media page in v7 had a nested content data type property that is now a nested content

I can see that there are migrators out there like NestedToBlockListMigrator - it's really great. Now I am wondering if it's possible to create a MigrationProfile where we say: 

```
if(doctype1)
{
    UseMigrator1()
}
else if(doctype2)
{
    UseMigrator2()
}
```

Is that something that is currently available ?

**Comment by KevinJump** [comment, marked as answer]
Hi, 

Yeah at the moment thats not possible, but its a good feature request ! 

I suspect we can probably do something where we have some migrator by propertyAlias then you could be that granular. 

Kevin

**Reply by giuunit** [reply]
Hi Kevin, could you elaborate regarding this approach ?
